nulib/dc-nextjs
# Remove all references to useFetchApi hook

## What does this do
Removes a troublesome hook `useFetchApiData()` previous used to get API /search endpoint data.   

## How to test
Go to the search page and interact with facets/filtering, etc.  Everything should behave as expected in the UI.
